e2e-tests/src/main/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java

```diff
@@ -732,6 +735,44 @@ public void searchQuery_should_require_searchDataAndSearchQuery_scopes() throws
                 .header("WWW-Authenticate", containsString("error=\"insufficient_scope\""));
     }
 
+    @Test
+    public void showSchemasFromCatalog_should_return_schemas() throws IOException {
+        DataConnectRequest query = new DataConnectRequest("SHOW SCHEMAS FROM " + showSchemaForCatalogName);
+        log.info("Running query {}", query);
+
+        Table result = dataConnectApiRequest(Method.POST, "/search", query, 200, Table.class);
+        result = dataConnectApiGetAllPages(result);
+
+        if (result.getData() == null) {
+            throw new RuntimeException("Expected results for query " + query.getQuery() + ", but none were found.");
+        }
```

Comment:
Same for line 765-768

Reply:
Hmm I now see that the e2e tests in this project do follow this pattern of throwing a RuntimeException if there are no results in the query rather than just asserting that its not null. So I guess its fine but I feel like doing the following is better as you don't need the `if` check - 
```
assertThat("Expected results for query " + query.getQuery() + ", but none were found.", query, not(nullValue())
```